test/hotspot/jtreg/vmTestbase/nsk/jvmti/Deallocate/dealloc001/dealloc001.cpp

```diff
@@ -24,7 +24,7 @@
 #include <stdio.h>
 #include <string.h>
 #include "jvmti.h"
-#include "agent_common.h"
+#include "agent_common.hpp"
 #include "JVMTITools.h"
```

Comment:
Why don't you change all of these together?  It's the same or similar scrolling.

Reply:
As I said in our private chat, I'll look at doing that for at least some of the remainder.  I already had this one
queued up, tested, and ready to submit the PR when you made that suggestion.  But avoiding that scrolling
is why I described how those updates were done.  If you believe the (implicit) suggestion in the PR description,
you don't need to look at them at all, and can just find the renamed file in the file list at the left in the review
window (so much less scrolling) and go look at it.

Reply:
Sure, but can you do the rest at the same time since they're in mostly the same files?